src/taskpane/taskpane.js

```diff
@@ -3,8 +3,7 @@
  * See LICENSE in the project root for license information.
  */
 
-/* global require */
-const excel = require("./excel");
-const outlook = require("./outlook");
-const powerpoint = require("./powerpoint");
-const word = require("./word");
+import "./excel";
```

Comment:
For javascript, shouldn't be converting require() to import here.

Reply:
The linter complains about using require saying the const values are assigned but not used.

Reply:
You can have the require() call without assigning it to the variable.